### PR TITLE
Release Chinese and standardize language strings

### PR DESCRIPTION
**Pull request by kluo**
@stvstnfrd @caesar2164

### CODE REVIEW

File: playbooks/roles/edxapp/defaults/main.yml [hunk 1]
```diff
@@ -104,10 +104,11 @@ EDXAPP_HELP_MODAL_LINKS:
...
 EDXAPP_LANGUAGES:
-  - ['ar', 'Arabic']
-  - ['en', "English"]
-  - ['es-419', "Espa\u00f1ol"]
-  - ['vi', 'Vietnamese']
+  - ['ar', 'العربية']
+  - ['en', 'English']
+  - ['es-419', 'Español (Latinoamérica)']
+  - ['vi', 'Tiếng Việt']
+  - ['zh-cn', '中文 (简体)']
```
Comment (caesar2164) [inline playbooks/roles/edxapp/defaults/main.yml]: is that: "Chinese (Mandarin)"?

Comment (kluo) [reply playbooks/roles/edxapp/defaults/main.yml]: No it says Chinese (Simplified)

Comment (caesar2164) [reply playbooks/roles/edxapp/defaults/main.yml]: oh ok.

btw, I love how "Simplified" uses more complicated characters 😁

Comment (stvstnfrd) [reply playbooks/roles/edxapp/defaults/main.yml]: Just curious, what's the reason for changing the second item in the list?

Comment (kluo) [reply playbooks/roles/edxapp/defaults/main.yml]: I just took the pairings from https://github.com/Stanford-Online/edx-platform/blob/master/lms/envs/common.py#L862
This also makes it consistent with edx.org account settings page.

Comment (stvstnfrd) [reply playbooks/roles/edxapp/defaults/main.yml]: Cool.

**Pull request merged**